DTS-STN/Alpha-Site
# Add Apollo gQL client function, project page fetches from AEM

# Description

[SCL-702 - Alter the GraphQL client function](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-702)

This PR addresses a bit more than just the above task as I've also replaced a lot of the old code within `projects.js` in order for it to fetch its data from AEM instead of Strapi. The projects content fragment model is still incomplete (FR fields, id field), so this page will need to be updated once the model (and fragments) have been completed.


## Test Instructions

1. DM me for the required env variable
2. Pull in branch
3. Type `yarn install --frozen-lockfile`
4. Type `yarn dev`
5. Navigate to `/projects`
6. Confirm that the project cards correspond to the fragments within AEM

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
